addons/controller_icons/ControllerIcons.gd

```diff
@@ -574,10 +574,20 @@ func _convert_key_to_path(scancode: int):
 			return "key/bracket_left"
 		KEY_BACKSLASH:
 			return "key/slash"
+		KEY_SLASH:
+			return "key/forwardslash"
 		KEY_BRACKETRIGHT:
 			return "key/bracket_right"
-		KEY_ASCIITILDE:
+		KEY_ASCIITILDE, KEY_QUOTELEFT:
```

Comment:
`KEY_QUOTELEFT` should be mapped to a dedicated \` character instead. Could you create an icon specific to it as well?